src/utils.js

```diff
@@ -53,7 +53,7 @@ export const getComponentValue = (inputValue, components, key) => {
       return inputValue ? 'Ja' : 'Nee';
     } else if (component.type === "select") {
       const obj = component.data.values.find(obj => obj.value === inputValue);
-      return obj ? obj.label : '';
+      return obj ? obj.label : inputValue;
```

Comment:
This change is just so we see some information in the summary page.  It will be improved in https://github.com/open-formulieren/open-forms/issues/597